plugin/src/main/java/com/iridium/iridiumcore/utils/ItemStackUtils.java

```diff
@@ -132,17 +133,12 @@ private static ItemStack setHeadData(String headData, ItemStack itemStack) {
         if (IridiumCore.isTesting()) return itemStack;
         if (headData == null) return itemStack;
 
-        NBTItem nbtItem = new NBTItem(itemStack);
-        NBTCompound skull = nbtItem.addCompound("SkullOwner");
-        if (supports) {
-            skull.setUUID("Id", getHeadDataUUID(headData));
-        } else {
-            skull.setString("Id", getHeadDataUUID(headData).toString());
-        }
-
-        NBTListCompound texture = skull.addCompound("Properties").getCompoundList("textures").addCompound();
-        texture.setString("Value", headData);
-        return nbtItem.getItem();
+        NBT.modify(itemStack, readWriteItemNBT -> {
+            if (supports) readWriteItemNBT.getOrCreateCompound("SkullOwner").setUUID("Id", getHeadDataUUID(headData));
+            else readWriteItemNBT.getOrCreateCompound("SkullOwner").setString("Id", getHeadDataUUID(headData).toString());
```

Comment:
```suggestion
            if (supports) {
                readWriteItemNBT.getOrCreateCompound("SkullOwner").setUUID("Id", getHeadDataUUID(headData));
            } else {
                 readWriteItemNBT.getOrCreateCompound("SkullOwner").setString("Id", getHeadDataUUID(headData).toString());
             }
```

Can we keep to this formatting on if/else statements please?